gis-data-sharing/open-data/index.md

```diff
@@ -6,16 +6,53 @@ author:
   display_name: Michael M. Foulger
   email: michaelfoulger@utah.gov
 date: 2019-11-12 13:40:06
-title: SGID Open Data Sharing
+title: Sharing Data through the SGID Open Data Site
 categories: []
 ---
 
-The following list outlines how to share data through the [SGID Open Data](http://opendata.gis.utah.gov/) website.
+![Open-Data-Homepage]({% link images/od_homepage.png %}){: .flex .flex--center .outline }
 
-- Although not required, we like to have a user named with the department’s name or function to be the one user to share data. AGRC uses a user called `UtahAGRC` to share all SGID authoritative data. This allows users to quickly determine the source of the data and the potential value of the data.
-- For data to be shared within the SGID, we prefer to share single feature type feature layers. This means, one data type and theme within a feature layer. This allows the greatest flexibility for the end users of the data.
-- Having an accurate title (Using Utah as a first identifier and spaces between words), summary and description are needed to allow users to discover your data.
-- For tagging data to be included within the ArcGIS Online SGID we start general and work to specific following the format `Agency/Department Name`, `SGID`, `ISO Category`, `other terms relevant to data usage`. We omit tags already contained in the layer title. Using the layer with the title `County Boundaries` as an example, we use `AGRC` and `SGID` as suggested tags. We chose `AGRC` since we want to include the agency and or department name. We chose `SGID` for inclusion in the SGID Open Data website. We omitted `County` and `Boundaries` since they are contained in the layer title. Please refer to [Using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.
-- Please make sure that the data is shared publicly with everyone, and include the appropriate SGID Group to share with. This will allow us to make your data available through the SGID Open Data website.
-- If you are the steward of the data and the data is considered authoritative, you can mark it as `Authoritative` so users will be able to tell at a glance that it is good data.
-- Please check the box to `Allow others to export to different formats` located at the bottom of the Settings page for the data to give the most flexibility for data usage.
+Does your department or division have authoritative datasets hosted in your own ArcGIS Online organization? Do you want to make them easy to find by anyone searching for Utah GIS Data? Share them on the [SGID Open Data website!](https://opendata.gis.utah.gov/) This makes them them easily discoverable while giving you full control over when they are updated.
```

Comment:
```suggestion
Does your department or division have authoritative datasets hosted in your own ArcGIS Online organization that are important to broadcast publicly? Do you want to make them easy to find by anyone searching for Utah GIS Data? Share them on the [SGID Open Data website!](https://opendata.gis.utah.gov/) This makes them them easily discoverable while giving you full control over when they are updated.
```

Reply:
I think I see where you're going with this, but I think the next sentence covers the public availability bit. I'm aiming for a logical progression: Data in a separate AGOL org? Check. Want to make it easy to find? Check. Then, share with open data.